src/OrleansSQLUtils/Storage/DbStoredQueries.cs

```diff
@@ -192,14 +192,18 @@ internal static ReminderEntry GetReminderEntry(IDataRecord record)
             /// </summary>
             private static string TryGetSiloName(IDataRecord record)
             {
-                for (var i = 0; i < record.FieldCount; ++i)
```

Comment:
It was erroring since the method was looking for the column in case sensitive way.  I changed it to use GetOrdinal() so that SiloName worked the same way as the other columns which use GetOrdinal() under the covers.  GetOrdinal() first does a case-sensitive search and if the column isn't found, it does a case-insensitive search.  One option is to duplicate this behavior so that it doesn't throw exceptions. 
Another option is to return the columns with the upper-lower case originally expected.  But just out of curiosity, why does this column work differently than the others?  ie "TryGet..."